File: Examples/LayoutGridExample.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.PageElements;

namespace DynamicPDFCoreSuite.Examples
{
    class LayoutGridExample
    {

        public static void Run()
        {
            Document document = new Document();
            Page page = new Page();
            document.Pages.Add(page);

            LayoutGrid grid = new LayoutGrid(LayoutGrid.GridType.Decimal);
            page.Elements.Add(new Rectangle(50, 50, 50, 50, 2));
            page.Elements.Add(grid);

            document.Draw(Util.GetPath("Output/layout-grid-output.pdf"));
        }
    }
}

File: dynamicpdf-vbnet-examples/Examples/LayoutGridExample.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.PageElements

Namespace DynamicPDFCoreSuite.Examples
    Class LayoutGridExample

        Public Shared Sub Run()
            Dim document As New Document()
            Dim page As New Page()
            document.Pages.Add(page)

            Dim grid As New LayoutGrid(LayoutGrid.GridType.Decimal)
            page.Elements.Add(New Rectangle(50, 50, 50, 50, 2))
            page.Elements.Add(grid)

            document.Draw(Util.GetPath("Output/layout-grid-output.pdf"))
        End Sub

    End Class
End Namespace
